readme.md

```diff
@@ -1,3 +1,70 @@
-# Pine Icon Libary
+# Pine Icon Library
 
-This is the Icon Library for Kajabi's Pine Design System.
+This is the Icon Library for Kajabi's Pine Design System. It provides a comprehensive set of SVG icons as web components that can be easily integrated into your projects.
+
+[![npm version](https://badge.fury.io/js/%40pine-ds%2Ficons.svg)](https://badge.fury.io/js/%40pine-ds%2Ficons)
+
+## Features
+
+- SVG icons optimized for web use
+- Web Components for easy integration
+- Customizable icon colors and sizes
+
+## Installation
+
+```bash
+npm install @pine-ds/icons
+```
+
+## Usage
+
+```html
+<pds-icon name="icon-name"></pds-icon>
+```
+
+## Local Development
+
+### Prerequisites
+
+- Node.js (v22.7.0 or higher)
+- npm (comes with Node.js)
+
+### Setup
+
+1. Clone the repository:
+```bash
+git clone https://github.com/Kajabi/pine-icons.git
+cd pine-icons
+```
+
+2. Install dependencies:
+```bash
+npm ci
+```
+
+3. Start the development server:
+```bash
+npm start
+```
+
+The development server will run on port 7200.
+
```

Comment:
This is probably great for `Developer setup`; do you think we should tailor this to how to use the library in a consuming app?